src/features/grants/utils/query-builder.ts

```
@@ -0,0 +1,137 @@
+  }
+
+  if (context === 'sponsor' && sponsor) {
+    const sponsorKey = sponsor.toLowerCase();
+
+    const sponsorInfo = exclusiveSponsorData[sponsorKey];
+
+    where.sponsor = {
+      name: sponsorInfo?.title,
+    };
+
+    if (!!sponsorInfo?.showPrivates) {
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)